exchanges/btcmarkets/btcmarkets_websocket.go

```diff
@@ -185,9 +185,12 @@ func (b *BTCMarkets) wsHandleData(respRaw []byte) error {
 		}
 		return nil
 	case tradeEndPoint:
```

Comment:
🗻 Out-Of-Scope, for another PR, but I just realised the size of this func, and it all being in wsHandleData
Needs breaking out into functions at some point.
Not high priority, just a chore, but worth mentioning.
Resolving.